### PR TITLE
docs: add doc content table and example in readme

### PR DESCRIPTION
**Pull request by aripiprazole**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,46 @@
 
 Welcome to the Woovi Java SDK! This SDK provides convenient access to the Woovi REST API, allowing you to easily integrate Woovi's video upload and management services into your Java applications.
 
+## Docs
+
+How do I use it? You can have a look in the following example:
+
+```java,no
+ApiClient defaultClient = Configuration.getDefaultApiClient();
+defaultClient.setBasePath("https://api.woovi.com");
```
Comment (edumaciel10) [inline README.md:11]: Always i need send it?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,46 @@
 
 Welcome to the Woovi Java SDK! This SDK provides convenient access to the Woovi REST API, allowing you to easily integrate Woovi's video upload and management services into your Java applications.
 
+## Docs
+
+How do I use it? You can have a look in the following example:
+
+```java,no
+ApiClient defaultClient = Configuration.getDefaultApiClient();
```
Comment (edumaciel10) [inline README.md:10]: Where come Configuration and ApiClient?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,46 @@
...
+defaultClient.setBasePath("https://api.woovi.com");
+
+// Configure API key authorization: AppID
+ApiKeyAuth AppID = (ApiKeyAuth) defaultClient.getAuthentication("AppID");
```
Comment (edumaciel10) [inline README.md:14]: why need force type here?

File: README.md (hunk 2 above)
Comment (aripiprazole) [reply README.md:10]: #11

File: README.md (hunk 3 above)
Comment (aripiprazole) [reply README.md:14]: `getAuthentication` returns `Authentication` class, and it needs to be `ApiKeyAuth` to set the App ID

**Pull request merged**